docs/en-us/cloneset.md

```diff
@@ -349,9 +439,26 @@ spec:
     paused: true
 ```
 
-### Lifecycle hook
+### Pre-download image for in-place update
+
+**FEATURE STATE:** Kruise v0.9.0
 
-Since v0.6.1, kruise has supported lifecycle hook for CloneSet.
+If you have enabled the `PreDownloadImageForInPlaceUpdate` feature-gate during [Kruise installation or upgrade](./installation.html#optional%3A-feature-gate),
+CloneSet controller will automatically pre-download the image you want to update to the nodes of all old Pods.
+It is quite useful to accelerate the progress of applications upgrade.
+
+The parallelism of each new image pre-downloading by CloneSet is `1`, which means the image is downloaded on nodes one by one.
+You can change the parallelism using the annotation on CloneSet:
```

Comment:
add suggestion about the parallelism setting such as

Note that the parallelism should be set according to the capability of image registry, for registries with more bandwidth and P2P image downloading ability, a larger parallelism can speed up the pre-download process.